exchanges/okx/okx_websocket.go

```diff
@@ -236,55 +235,19 @@ var subscriptionNames = map[string]string{
 }
 
 // WsConnect initiates a websocket connection
-func (e *Exchange) WsConnect() error {
```

Comment:
At this kind of scale its harder to pinpoint exactly why this is failing as much as it is:
<img width="1727" height="975" alt="image" src="https://github.com/user-attachments/assets/b04a4a4e-977e-4c45-8597-ff8eb4b50297" />
This seems to specifically affect SPOT orderbooks.
I'm happy for this to be raised as a separate issue that is starting to show as a result of this scaling.

Reply:
Deferred to: #2136